Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -51,6 +57,11 @@ @interface WebViewController () <LanguageSelectionDelegate, FetchFinishedDelegat
 
 @property (strong, nonatomic) WMFArticlePopupTransition* popupTransition;
 
+@property (nonatomic, strong) NSMutableArray* nativeTitleLabelModelsArray;
```

Comment:
Mostly done w/this last commit. Updating it now to use protocol and/or delegate bits for communication.